conpr/08_LE_Tasks/src/le/fj/Sum.java

```diff
@@ -35,7 +35,7 @@ protected Integer compute() {
     
     public static void main(String[] args) {
         Random rnd = new Random();
-        int SIZE = 500000000;
+        int SIZE = 40000000;
```

Comment:
SIZE = 500,000,000 would give an expected sum of 500,000,000 * 49.5 = 24,750,000,000, which is well above the upper bound for 32-bit integers. Reducing SIZE to 40,000,000 makes the expected sum 40,000,000 * 49.5 = 1,980,000,000, which will not cause integer overflow problems.